app/server/tests/functions/authFunctions.js

```diff
@@ -71,23 +71,37 @@ class AuthFunctions{
         return deleteResponse;
     }
 
```

Comment:
^^ loginInsecure includes .set('Credentials', 'Include') but login does not. Also, is 'Include' case sensitive? Worth a try.